ui/components/src/EChart.tsx

```diff
@@ -50,7 +50,7 @@ const mouseEvents = [
   'contextmenu',
 ] as const;
 
-export type MouseEventName = typeof mouseEvents[number];
+export type MouseEventName = (typeof mouseEvents)[number];
```

Comment:
One of the other deps installed led to an update to prettier, so there's a few minor changes like this in here. Seems harmless to me, but let me know if you think we need to pin prettier at a particular version.